### PR TITLE
Remove unneeded `mut`

### PR DESCRIPTION
**Pull request by p-l-**
This removes a compilation warning.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[![Join the chat at Gitter](https://badges.gitter.im/ivre/ivre.svg)](https://gitter.im/ivre/ivre)
```
Comment (Frky) [inline README.md:1]: Hm this should be in another PR no?

Comment (p-l-) [reply README.md:1]: Yep sorry pushed too fast

**Review by Frky — APPROVED** [review]
